bis-rules/package.json

```diff
@@ -68,10 +68,10 @@
     "extends": "plugin:@itwin/itwinjs-recommended",
     "rules": {
       "prefer-template": "off",
-      "dot-notation": "off",
+      "@typescript-eslint/dot-notation": "off",
       "@typescript-eslint/await-thenable": "off",
```

Comment:
lets remove this rule suppression as well.  It's not so catastrophic but awaiting something which does not require it is a good sign that something else may be wrong

Reply:
I have removed it and fixed the lint issues accordingly. 

At some places we required to disable it e.g `property!["_customAttributes"] = propertyCAMap;` due to 'Property '_customAttributes' is private and only accessible within class 'Property''

Reply:
Done